config/locales/en.yml

```diff
@@ -31,4 +31,7 @@
 
 en:
   hello: "Hello world"
+  roles:
+    partner: "Regular"
```

Comment:
This seems an odd string to associate with the partner role... is this trying to do something a weird way?

Reply:
Hmm been a while, I don't remember. Lemme check whats this about.

Reply:
Removed! I think this was a typo.